base/test_views.py

```diff
@@ -11,17 +14,17 @@ def tearDown(self):
         self.browser.quit()
 
     def test_home(self):
-        self.browser.visit('http://localhost:8000')
+        self.browser.visit(test_url)
```

Comment:
The full url, should not be generated this way, as this test will break if we decide to change the way visit those sites.  `self.life_server_url + reverse('home')` should be the way to do this.

The same idea applies to the rest of the urls to be visited on this page.